bundler.d/test.rb

```diff
@@ -4,7 +4,7 @@
   gem 'test-unit'
   gem 'benchmark-ips'
   gem 'ruby-prof', '< 1.4'
-  gem 'rubocop', '~> 0.89.0'
+  gem 'rubocop', '~> 1.28.0'
```

Comment:
We can update to the latest version:
```suggestion
  gem 'rubocop', '~> 1.55.0'
```